docs/developer-docs/integrations/sns/integrate-sns/ledger-integration.md

```diff
@@ -0,0 +1,111 @@
+# SNS Ledger Canister
```

Comment:
I think this is missing as header in the preview https://deploy-preview-710--icportal.netlify.app/docs/current/developer-docs/integrations/sns/integrate-sns/ledger-integration . There is only the SNS Index Canister.

Reply:
Updated the sidebar

Reply:
This is intentional as otherwise there would be an extra 'click' for this dropdown. A way to resolve it is to have an 'overview' page for general SNS integrations (describing frontend, ledger, etc). Then both the ledger and index pages could appear in the sidebar